pydantic_ai_slim/pydantic_ai/models/openai.py

```diff
@@ -151,13 +151,28 @@
     '3:2': '1536x1024',
 }
 
+_OPENAI_IMAGE_SIZE_VALUES: tuple[Literal['auto', '1024x1024', '1024x1536', '1536x1024'], ...] = (
```

Comment:
We can make `Literal['auto', '1024x1024', '1024x1536', '1536x1024']` a type, and then use it in the return signature for `_resolve_openai_image_generation_size` _and_ at runtime using `_utils.get_union_args` (or if that doesn't work `typing.get_args`)